src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+
+public interface ReportService {
+    String createReport(Storage storage);

```

Comment:
we can access storage directly
```suggestion

public interface ReportService {
    String createReport();

```